test/Ops-multiModule.test.ts

```diff
@@ -0,0 +1,215 @@
+import { Signer } from "@ethersproject/abstract-signer";
+import { expect } from "chai";
+import {
+  encodeResolverArgs,
+  encodeTimeArgs,
+  fastForwardTime,
+  getTaskId,
+  getTimeStampNow,
+  Module,
+  ModuleData,
+} from "./utils";
+import hre = require("hardhat");
+const { ethers, deployments } = hre;
+import {
+  Ops,
+  CounterWithWhitelist,
+  CounterResolver,
+  TaskTreasuryUpgradable,
+  ResolverModule,
+  TimeModule,
+  ProxyModule,
+  SingleExecModule,
+  OpsUserProxyFactory,
+  OpsUserProxy,
+} from "../typechain";
+
+const execFuncSig =
+  "exec(address,address,bytes,(uint8[],bytes[]),uint256,address,bool,bool)";
+
+const GELATO = "0x3caca7b48d0573d793d3b0279b5f0029180e83b6";
+const ETH = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
+const ZERO_ADD = ethers.constants.AddressZero;
+const FEE = ethers.utils.parseEther("0.1");
+const INTERVAL = 7 * 60;
+
+describe("Ops multi module test", function () {
```

Comment:
We should also add a test scenario of creating a task with no modules at all
ie: task with predefined input data that executes whenever possible